server/api/usersAtEvents.js

```diff
@@ -32,3 +32,19 @@ router.put('/', canOnlyBeUsedBy('admin', 'leader'), async (req, res, next) => {
     next(err)
   }
 })
+
+router.put(
```

Comment:
probably should be a delete route?

Reply:
like .delete('/:eventId') instead of at a /remove endpoint.

Reply:
It's interesting because we are not removing an event but an association of a user to an event. Delete(:eventId) would make it seem like we are removing the event itself

Reply:
The problem here is that router.delete creates a req.body that's an empty object. 
Researched this for 2 days, req has the correct headers and everything else is the same as the other route types besides missing the body. 

Considering the time crunch, Permission to just use a PUT here?

Reply:
Nvm fixed. Weird Special syntax